lib/routes/charge/items/item_avatar_picker.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:badges/badges.dart';
+import 'package:breez/theme_data.dart';
+import 'package:breez/utils/icon_map.dart';
+import 'package:breez/widgets/back_button.dart' as backBtn;
+import 'package:flutter/material.dart';
+
+import 'item_avatar.dart';
+
+class ItemAvatarPicker extends StatefulWidget {
+  final String itemImage;
+  final Function(String selectedImage) onImageSelected;
+  final String itemName;
+
+  ItemAvatarPicker(this.itemImage, this.onImageSelected, {this.itemName});
+
+  @override
+  State<StatefulWidget> createState() {
+    return ItemAvatarPickerState();
+  }
+}
+
+class ItemAvatarPickerState extends State<ItemAvatarPicker> {
+  Map<String, IconData> iconMap = IconMap().iconMap;
```

Comment:
Can we do this static? so it won't be loaded every time?

Reply:
[Done](https://github.com/breez/breezmobile/pull/319/commits/e7f34f15e5c04d82fe1331ebe3d39564a487dd68)